typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -320,7 +320,7 @@ export const ethereum: ChainMetadata = {
   nativeToken: etherToken,
   protocol: ProtocolType.Ethereum,
   rpcUrls: [
-    { http: 'https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161' },
+    { http: 'https://ethereum.publicnode.com' },
```

Comment:
So the key-ed infura endpoint doesn't support fork testing but this public one does? That's surprising

Reply:
I can revert this, I thought it was weird to have the keyed endpoint here